internal/devconfig/packages_test.go

```diff
@@ -172,42 +159,31 @@ func TestJsonifyConfigPackages(t *testing.T) {
 
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			config := &Config{}
-			if err := json.Unmarshal([]byte(testCase.jsonConfig), config); err != nil {
-				t.Errorf("unexpected error: %v", err)
-			}
-			if !reflect.DeepEqual(config.Packages, testCase.expected) {
-				t.Errorf("expected: %v, got: %v", testCase.expected, config.Packages)
-			}
-
-			marshalled, err := json.Marshal(config)
+			config, err := loadBytes([]byte(testCase.jsonConfig))
 			if err != nil {
-				t.Errorf("unexpected error: %v", err)
+				t.Errorf("load error: %v", err)
 			}
-			if string(marshalled) != testCase.jsonConfig {
-				t.Errorf("expected: %v, got: %v", testCase.jsonConfig, string(marshalled))
+			if diff := diffPackages(t, config.Packages, testCase.expected); diff != "" {
+				t.Errorf("got wrong packages (-want +got):\n%s", diff)
 			}
 
-			// We also test cuecfg.Marshal because elsewhere in our code we rely on it.
-			// While in this PR it is now a simple wrapper over json.Marshal, we want to
-			// ensure that any future changes to that function don't break our code.
-			marshalled, err = cuecfg.Marshal(config, ".json")
+			got, err := hujson.Minimize(config.Bytes())
 			if err != nil {
-				t.Errorf("unexpected error: %v", err)
-			}
-			// We need to pretty-print the expected output because cuecfg.Marshal returns
-			// the json pretty-printed.
-			expected := &bytes.Buffer{}
-			if err := json.Indent(expected, []byte(testCase.jsonConfig), "", cuecfg.Indent); err != nil {
-				t.Errorf("unexpected error: %v", err)
+				t.Fatal(err)
 			}
-			if string(marshalled) != expected.String() {
-				t.Errorf("expected: %v, got: %v", testCase.jsonConfig, string(marshalled))
+			if string(got) != testCase.jsonConfig {
+				t.Errorf("expected: %v, got: %v", testCase.jsonConfig, string(got))
```

Comment:
It should be checking both. It unmarshals from `testCase.jsonConfig` and then marshals back with `got, err := hujson.Minimize(config.Bytes())` on line 170. It doesn't do `cuecfg.Marshal` anymore because the new code isn't using `cuecfg`.